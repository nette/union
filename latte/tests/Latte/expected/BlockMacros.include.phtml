<?php
%A%

final class Template%a% extends Latte\Runtime\Template
{

	public function main(): array
	{
%A%
		/* line %d% */
		$this->createTemplate(('subdir/include1.latte' . ''), ['localvar' => 10] + $this->params, 'include')->renderToContentType(function ($s, $type) {
			$ʟ_fi = new LR\FilterInfo($type);
			return LR\Filters::convertTo($ʟ_fi, 'html', $this->filters->filterContent('indent', $ʟ_fi, $s));
		});
		return get_defined_vars();
	}

}
