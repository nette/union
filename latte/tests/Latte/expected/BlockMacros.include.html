
	<p>Included file #1</p>

	<p>Included file #2 (20, &lt;i&gt;Hello&lt;/i&gt;)</p>

	Parent: include1.latte/include

	<p>Included file #3 (10, &lt;i&gt;Hello&lt;/i&gt;)</p>

	<textarea>
pre
</textarea>

	Parent: BlockMacros.include.latte/include
